app/Domain/Session.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Notes\Domain;
+
+use Notes\Exception\ModelNotFoundException as ModelNotFoundException;
+use Notes\Mapper\Session as SessionMapper;
+use Notes\Model\Session as SessionModel;
+use Notes\Domain\User as UserDomain;
+use Notes\Validator\InputValidator as InputValidator;
+
+class Session
+{
+    public function __construct()
+    {
+        $this->validator = new InputValidator();
+    }
+    
+    public function create($userModel, $sessionModel)
+    {
+        try {
+            $userDomain    = new UserDomain();
+            $userModelRead = $userDomain->readByUsernameandPassword($userModel);
+            if (!empty($userModelRead)) {
+                $sessionModel->setUserId($userModelRead->getId());
+                if ($this->validator->notNull($sessionModel->getUserId())
+                && $this->validator->validNumber($sessionModel->getUserId())) {
+                    $sessionMapper = new SessionMapper();
+                    $sessionModel  = $sessionMapper->create($sessionModel);
+                    return $sessionModel;
+                }
+            }
+        } catch (Notes\Exception\ModelNotFoundException $e) {
```

Comment:
We have included ModelNotFoundException using use statement. so we don't need to use full namespace path.